packages/editor-ui/src/views/NodeView.vue

```diff
@@ -510,6 +516,36 @@ export default mixins(
 					});
 				}
 			},
+			orientNodes(workflowNodes: INode[]) {
```

Comment:
suggestion - could move this to nodes helpers since it does not require state. can rename to `getFixedNodesList` or something like that to be more clear